### PR TITLE
feat(interfaces): adds user interface

### PR DESCRIPTION
**Pull request by hannahu**

### CODE REVIEW

File: packages/interfaces/src/user.interface.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+export interface UserGroup {
+    name: string;
+    id: number;
+    description: string;
+}
+
+interface UserCoworker {
+    id: number;
+    limetype: string;
+}
```
Comment (jgroth) [inline packages/interfaces/src/user.interface.ts:52]: These could use some doc blocks as well :slightly_smiling_face:

**Review by jgroth — COMMENTED** [review]
Looks good! Some missing documentation and there are a few lint errors as well :slightly_smiling_face:

**Comment by hannahu** [comment]
> Looks good! Some missing documentation and there are a few lint errors as well slightly_smiling_face

Oki!!

File: packages/interfaces/src/user.interface.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
+export interface User {
+    /**
+     * The name of the user
+     */
+    username: string;
+
```
Comment (Befkadu1) [inline packages/interfaces/src/user.interface.ts:6]: do we need to have `isAdmin: boolean` or `admin: boolean` ?

Comment (hannahu) [reply packages/interfaces/src/user.interface.ts:6]: This is what we get from the endpoint, what you suggest doesn't exist in the user data :)

Comment (Befkadu1) [reply packages/interfaces/src/user.interface.ts:6]: Aha I confused it with Session

**Review by Befkadu1 — DISMISSED** [review]

**Review by jgroth — APPROVED** [review]

**Pull request merged**